CONTRIBUTING.org

```diff
@@ -200,6 +204,12 @@ version is detected by running ~STABLE=t make recipes/<NAME>~.  The
 version detection can be adjusted by specifying ~:version-regexp~ in
 the recipe (see [[file:README.md#recipe-format][recipe format]] in the README).
 
+**** Test your recipe installs
+
+In your copy of the MELPA repository.  Run the installation check
```

Comment:
That's not a complete sentence, and the second is a run-on, but this paragraph seems unnecessary, anyway.  Instead, let's add a paragraph after the following one that says something like:
```
To test a stable release:

1.  Build the recipe by running ~STABLE=t make recipes/<NAME>~.
2.  Install the package from the ~packages-stable~ directory.
```

Reply:
Incorporated into 3b7c979.